lib/src/reddit.dart

```diff
@@ -620,6 +620,7 @@ class Reddit {
           'Cannot make requests using unauthenticated client.');
     }
     final path = Uri.https(defaultOAuthApiEndpoint, api);
+    params ??= {'raw_json': '1'}; // only on null
```

Comment:
Currently only set to add the param if the 'params' parameter is null. This should make it so that certain calls can override this.